client/lib/DomState.ts

```diff
@@ -0,0 +1,13 @@
+import IDomState, { IDomStateAssertions } from '@ulixee/hero-interfaces/IDomState';
+
+export default class DomState implements IDomState {
+  public name?: string;
+  public url?: string | RegExp;
+  public allTrue: (options: IDomStateAssertions) => void;
+
+  constructor(options: IDomState) {
+    this.name = options.name;
+    this.url = options.url;
+    this.allTrue = options.allTrue;
```

Comment:
Does allTrue feel like the right name?

Reply:
It feels a little strange, but I didn't have a better name. We could do something more like "map" syntax- like every, or all